pubsub/example_test.go

```diff
@@ -0,0 +1,21 @@
+package pubsub_test
+
+import (
+	"testing"
```

Comment:
no, but I've found that I like the packages providing a clean example of usage. For example https://github.com/go-kit/kit/blob/master/log/example_test.go